tests/test_hf_api.py

```diff
@@ -79,22 +78,9 @@
 
 logger = logging.get_logger(__name__)
 
-
-def repo_name(id=uuid.uuid4().hex[:6]):
-    return "my-model-{0}-{1}".format(id, int(time.time() * 10e3))
-
-
-def repo_name_large_file(id=uuid.uuid4().hex[:6]):
-    return "my-model-largefiles-{0}-{1}".format(id, int(time.time() * 10e3))
-
-
-def dataset_repo_name(id=uuid.uuid4().hex[:6]):
-    return "my-dataset-{0}-{1}".format(id, int(time.time() * 10e3))
-
-
-def space_repo_name(id=uuid.uuid4().hex[:6]):
-    return "my-space-{0}-{1}".format(id, int(time.time() * 10e3))
-
+dataset_repo_name = partial(repo_name, prefix="my-dataset")
+space_repo_name = partial(repo_name, prefix="my-space")
+large_file_repo_name = partial(repo_name, prefix="my-model-largefiles")
```

Comment:
Just a note on readability: in general we try not to use tools from `functools` as these tend to severely hinder readability. I think it's perfectly fine in that case as it's very clear, but just to keep in mind for the future.

Reply:
Understood, makes sense 👍